### PR TITLE
Added Quarkus and React to new-proj

### LINKED ISSUE #111: add quarkus and react to new proj
As a developer, I want to link the ready code parts together.
As a user, I do not want to use an unfinished software.

### PR DESCRIPTION
**Pull request by veikkahonkanen**
Resolves #111

### CODE REVIEW

File: src/commands/new-proj.ts [hunk 1]
```diff
@@ -150,6 +180,37 @@ async function initDefaultProject() {
...
+ * Inits a React project
+ */
+async function initReactProject() {
+  try {
+    const cmd: string = await CreateReact(projName);
+    await runExecSync(cmd);
+
+    const cmds: string[] = await CleanReact(folderPath, repoPath);
+    for (let i = 0; i < cmds.length; i++) {
+      (i <= 8) ? await runExecSync(cmds[i]) : await runExecSync(cmds[i], { cwd: `.${path.sep}resources` });
```
Comment (eetupur) [inline src/commands/new-proj.ts]: Are all commands really run in resources folder?

**Review by eetupur — COMMENTED** [review]
Found possibly questionable design.

File: src/commands/new-proj.ts (hunk 1 above)
Comment (veikkahonkanen) [reply src/commands/new-proj.ts]: Yes. See React Utils.

File: src/commands/new-proj.ts [hunk 2]
```diff
@@ -77,13 +80,40 @@ async function action() {
...
+        choices: [ "Yes", "No" ],
+        message: "Use Gradle?: "
+      });
+
+      if (!kotlinResult.kotlin || !gradleResult.gradle) {
+        kotlin = false;
+        gradle = false;
+      } else {
+        (kotlinResult.kotlin == "Yes") ? kotlin = true : kotlin = false;
+        (gradleResult.gradle == "Yes") ? gradle = true : gradle = false;
```
Comment (jarinykanen) [inline src/commands/new-proj.ts]:
```suggestion
        kotlin = (kotlinResult.kotlin == "Yes");
        gradle = (gradleResult.gradle == "Yes");
```

File: src/commands/new-proj.ts [hunk 3]
```diff
@@ -150,6 +180,37 @@ async function initDefaultProject() {
...
+ */
+async function initReactProject() {
+  try {
+    const cmd: string = await CreateReact(projName);
+    await runExecSync(cmd);
+
+    const cmds: string[] = await CleanReact(folderPath, repoPath);
+    for (let i = 0; i < cmds.length; i++) {
+      (i <= 8) ? await runExecSync(cmds[i]) : await runExecSync(cmds[i], { cwd: `.${path.sep}resources` });
+    }
```
Comment (jarinykanen) [inline src/commands/new-proj.ts]: Might be a bit easier to read like this
```suggestion
    for (let i = 0; i < cmds.length; i++) {
      if (i < 9) {
        await runExecSync(cmds[i]);
      } else {
        await runExecSync(cmds[i], { cwd: `.${path.sep}resources` });
      }
    }
```

**Review by jarinykanen — CHANGES_REQUESTED** [review]

**Review by eetupur — APPROVED** [review]

**Review by jarinykanen — APPROVED** [review]

**Pull request merged**